nextflow_schema.json

```diff
@@ -30,6 +30,14 @@
                     "help_text": "You will need to create a design file with information about the samples in your experiment before running the pipeline. Use this parameter to specify its location. It has to be a comma-separated file.",
                     "fa_icon": "fas fa-file-csv"
                 },
+                "workflow_mode": {
+                    "type": "string",
+                    "default": "full",
+                    "enum": ["full", "simplified"],
+                    "description": "Workflow mode: 'full' runs the complete pipeline with all optional analyses (BUSCO, IQTREE_PMSF, GENERAX_PER_FAMILY); 'simplified' runs a streamlined high-throughput workflow optimized for large datasets.",
+                    "help_text": "The 'full' mode (default) provides maximum flexibility and includes quality assessments (BUSCO) and detailed per-family GeneRax analyses. The 'simplified' mode uses a fixed toolchain (WITCH->CIAlign->FastTree) and skips optional steps for faster execution on large datasets.",
```

Comment:
Just another note about the above comment regarding full/simplified/zoogle workflow modes.